synapse/storage/databases/main/presence.py

```diff
@@ -73,6 +73,15 @@ def __init__(
             prefilled_cache=presence_cache_prefill,
         )
 
+        # Used by `_get_active_presence()`
+        self.db_pool.updates.register_background_index_update(
+            "presence_stream_not_offline_index",
+            index_name="presence_stream_state_not_offline_idx",
```

Comment:
Do we have a naming convention for background updates, indices and .sqls?

Reply:
not really! these names look good to me.